lib/src/launchpad.rs

```diff
@@ -136,72 +136,87 @@ impl LaunchpadMk2 {
     pub fn light_led(&mut self, led: &ColorLed) {
         // F0h 00h 20h 29h 02h 18h 0Ah <LED> <Colour> F7h
         // Message can be repeated up to 80 times.
-        self.light_leds(&[led])
+
+        // Not sure here on how to use it without a copy
```

Comment:
Feel free to add a dependency on this: https://docs.rs/ref_slice/1.1.1/ref_slice/fn.ref_slice.html